.github/workflows/android_ci.yml

```
@@ -0,0 +1,93 @@
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: ${{ runner.os }}-gradle-${{ hashFiles('gradle.properties', '**/*.gradle*', '**/gradle-wrapper.properties') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+
+      # Android SDK 캐싱
+      - name: Cache Android SDK
+        uses: actions/cache@v2
```

actionlint (1.7.4)

the runner of "actions/cache@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)